app/plugin.go

```diff
@@ -184,6 +184,10 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 
 	plugins := a.processPrepackagedPlugins(prepackagedPluginsDir)
 	pluginsEnvironment = a.GetPluginsEnvironment()
+	if pluginsEnvironment == nil {
```

Comment:
Would it make sense to just use the `env` created a few lines above and sidestep the unexpected shutdown?

Reply:
That was another option that I considered but I opted for returning early and avoid the extra time since getting a `nil` environment at that point can only mean that either something is very wrong or we are simply shutting down.
I wasn't sure it made sense to continue at that point but I guess both ways work.

Reply:
:+1: